etc/profile-m-z/zoom.profile

```diff
@@ -10,7 +10,9 @@ noblacklist ${HOME}/.zoom
 
 include disable-common.inc
 include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
+include disable-passwdmgr.inc
 include disable-programs.inc
```

Comment:
Can we add `disable-shell` and `disable-xdg`?

Reply:
I have firejail version 0.9.62 so I can't test disable-shell. I can add disable-xdg, looks harmless

Reply:
You can copy it. disabel-shell works in 0.9.62 too.

(If `firejail --blacklist=/bin/bash zoom` fails you can skip it).

Reply:
tested, zoom fails with disable-shell